templates/service_templates/redis.service.erb

```diff
@@ -1,17 +1,24 @@
 [Unit]
-Description=Advanced key-value store for <%= @title %>
+Description=Redis Advanced key-value store for instance <%= @title %>
 After=network.target
+After=network-online.target
+Wants=network-online.target
 
 [Service]
 RuntimeDirectory=redis
 RuntimeDirectoryMode=2755
 Type=forking
```

Comment:
If it's supervised, I think the type should also be `notify` for optimal integration.

Reply:
Sure, will do.